client/api/__mocks__/api.js

```diff
@@ -0,0 +1,5 @@
+export function addFood(food) {
```

Comment:
Not sure why this is in a mock folder but isn't mocking anything. Should this be in `tests/client/_mocks_` like this?

https://github.com/don-smith/reka/blob/master/tests/client/mocks/api.js

Reply:
Done as you command